whotracksme/website/plotting/plots.py

```diff
@@ -50,9 +53,18 @@ def profile_doughnut(values, labels, name, color_palette=False):
             )
         ]
     )
-    fig = dict(data=data, layout=layout)
-
-    return div_output(fig)
+    fig = go.Figure(data=data, layout=layout)
+    image_bytes = fig.to_image(format='svg', height=200, width=161)
+
+    return f'''
+    <div class="plotly-graph-div js-plotly-plot" style="height: 100%; width: 100%;">
+        <div class="plot-container plotly">
+        <div class="svg-container" style="position: relative; width: 161px; height: 200px">
+            {image_bytes.decode('utf-8')}
+        </div>
+        </div>
+    </div>
```

Comment:
This looks very similar to another snippet of code in `companies.py`. Maybe having a generic helper would make sense?